internal/bind/params.go

```diff
@@ -177,63 +244,82 @@ func toYdbParam(name string, value interface{}) (table.ParameterOption, error) {
 func Params(args ...interface{}) (params []table.ParameterOption, _ error) {
 	params = make([]table.ParameterOption, 0, len(args))
 	for i, arg := range args {
+		var err error
 		switch x := arg.(type) {
 		case driver.NamedValue:
-			if x.Name == "" {
-				switch xx := x.Value.(type) {
-				case *table.QueryParameters:
-					if len(args) > 1 {
-						return nil, xerrors.WithStackTrace(errMultipleQueryParameters)
-					}
-					xx.Each(func(name string, v types.Value) {
-						params = append(params, table.ValueParam(name, v))
-					})
-				case table.ParameterOption:
-					params = append(params, xx)
-				default:
-					x.Name = fmt.Sprintf("$p%d", i)
-					param, err := toYdbParam(x.Name, x.Value)
-					if err != nil {
-						return nil, xerrors.WithStackTrace(err)
-					}
-					params = append(params, param)
-				}
-			} else {
-				param, err := toYdbParam(x.Name, x.Value)
-				if err != nil {
-					return nil, xerrors.WithStackTrace(err)
-				}
-				params = append(params, param)
-			}
+			params, err = processNamedValue(params, x, i, len(args))
 		case sql.NamedArg:
-			if x.Name == "" {
-				return nil, xerrors.WithStackTrace(errUnnamedParam)
-			}
-			param, err := toYdbParam(x.Name, x.Value)
-			if err != nil {
-				return nil, xerrors.WithStackTrace(err)
-			}
-			params = append(params, param)
+			params, err = appendParam(params, x.Name, x.Value)
```

Comment:
it is named param too. Why case `case driver.NamedValue` translated to `processNamedValue`, but this case to common `appendParam`?